native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/sql-native/resource-config.json

```diff
@@ -0,0 +1,15 @@
+{
+  "resources":{
+  "includes":[
+    {
+      "pattern":"\\QMETA-INF/services/java.sql.Driver\\E"
+    }, 
+    {
+      "pattern":"\\Qorg/slf4j/impl/StaticLoggerBinder.class\\E"
```

Comment:
I think we can remove this resource class. Can you try building the image after removing this class?

Reply:
Fixed in [4263c4c](https://github.com/ballerina-platform/module-ballerina-sql/pull/563/commits/4263c4cf80dbb61c539e18ad8c99181bbc80ec4b)